api/v1alpha1/gateway_types.go

```diff
@@ -270,28 +270,22 @@ type GatewayStatus struct {
 type GatewayConditionType string
 
 const (
-	// ConditionNoSuchGatewayClass indicates that the specified GatewayClass
-	// does not exist.
-	ConditionNoSuchGatewayClass GatewayConditionType = "NoSuchGatewayClass"
-	// ConditionGatewayNotScheduled indicates that the Gateway has not been
-	// scheduled.
-	ConditionGatewayNotScheduled GatewayConditionType = "GatewayNotScheduled"
-	// ConditionListenersNotReady indicates that at least one of the specified
-	// listeners is not ready. If this condition has a status of True, a more
-	// detailed ListenerCondition should be present in the corresponding
-	// ListenerStatus.
-	ConditionListenersNotReady GatewayConditionType = "ListenersNotReady"
-	// ConditionInvalidListeners indicates that at least one of the specified
-	// listeners is invalid. If this condition has a status of True, a more
-	// detailed ListenerCondition should be present in the corresponding
-	// ListenerStatus.
-	ConditionInvalidListeners GatewayConditionType = "InvalidListeners"
-	// ConditionRoutesNotReady indicates that at least one of the specified
-	// routes is not ready.
-	ConditionRoutesNotReady GatewayConditionType = "RoutesNotReady"
-	// ConditionInvalidRoutes indicates that at least one of the specified
-	// routes is invalid.
-	ConditionInvalidRoutes GatewayConditionType = "InvalidRoutes"
+	// ConditionGatewayClassExists indicates that the specified GatewayClass exists.
+	ConditionGatewayClassExists GatewayConditionType = "GatewayClassExists"
+	// ConditionGatewayScheduled indicates that the Gateway has been scheduled.
+	ConditionGatewayScheduled GatewayConditionType = "GatewayScheduled"
```

Comment:
You have to be more specific here. "Scheduled" means what for the end user?
For stuff that is informational, e.g. automation cannot act on it, has no semantic API meaning, it's worth thinking about whether it requires its own constant or just put the information into the human readable form.

Reply:
> You have to be more specific here. "Scheduled" means what for the end user?

@bowei I tried to make minimal changes to the docs to support the intent of this PR, which is to reverse the polarity of the existing conditions. This was an existing condition. Do you prefer I make changes to the previously created condition types in this PR or have this PR strictly focus on reversing the polarity?